doc/dev/MAINTAINERS_AND_CONTRIBUTORS.md

```diff
@@ -0,0 +1,25 @@
+# Maintainers and Contributors
+
+This document lists the Maintainers and Returning Contributors of the Project.
+
+## Maintainers
+
+Maintainers may add themselves once approved by the existing Maintainers as described in the [Governance document](./GOVERNANCE.md). By adding your name to this list you are agreeing to abide by and uphold the Project governance documents and to abide by all of the Organization's policies, including the [code of conduct](../../CODE_OF_CONDUCT.md).
+
+| **NAME**         | **Handle** |
+| ---------------- | ---------- |
+| Mathieu Westphal | mwestphal  |
+| Michael Migliore | Meakk      |
+
+## Returning Contributors
```

Comment:
OK inactivity period and optional listing is specified here, but it wouldn't hurt to repeat it in the other file.